jakartaee-web-api/pom.xml

```diff
@@ -310,6 +238,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Workaround for Javadoc failures -->
```

Comment:
This is normal intended behaviour:

As documented (https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html) optional dependencies are not included transitional, so they need to decalred again, when used.